x/cosmwasmpool/cosmwasm/msg/module_query_msg.go

```diff
@@ -0,0 +1,27 @@
+// This file defines helpers for querying
+// the cosmwasm pool contract from the cosmwasm pool module.
+package msg
+
+import sdk "github.com/cosmos/cosmos-sdk/types"
+
+// CalcOutAmtGivenIn
+func NewCalcOutAmtGivenInRequest(tokenIn sdk.Coin, tokenOutDenom string, swapFee sdk.Dec) CalcOutAmtGivenInRequest {
+	return CalcOutAmtGivenInRequest{
```

Comment:
Just our of curiosity, these functions should always return the the same as the inputs for transmuter, right?

Reply:
yeah, that makes sense. My understanding of transmutter is that it just swaps tokens 1:1. So I guess these functions don't have anything to calculate: send 1 axlusdc, get 1 nobleusdc.